server/src/router.ts

```diff
@@ -12,9 +12,17 @@ const app = express();
 
 const httpServer = http.createServer(app);
 const io = new socketio.Server(httpServer);
+const IMAGE_SHOWN_SECONDS = 60
 
 const clientPositions: { [clientId: string]: { x: number; y: number } } = {};
 
+export interface IBackgroundState {
+  isShowingImage: boolean;
```

Comment:
isShowingImage seems redundant, if currentBackground is undefined then we know we are not showing an image, similarly if currentBackground has a value, we know we are showing an image

we could always check by doing `const isShowingImage = !!backgroundState.currentBackground` which would turn the string into a boolean value, if currentBackground is undefined then boolean is false, if defined boolean is true